assets/js/template-helpers/useTranslation.js

```diff
@@ -0,0 +1,70 @@
+export default function (string, translations, options) {
+  switch (string) {
+    case 'Bring up this help dialog':
+      return translations.bring_dialog
+    case 'callback':
+      return translations.callback
+    case 'Callbacks':
+      return translations.callbacks
+    case 'Contributing':
+      return translations.contributing
+    case 'extras':
+      return translations.extras
+    case 'Focus search bar':
+      return translations.focus_search_bar
+    case 'function':
+      return translations.function
+    case 'Functions':
+      return translations.functions
+    case 'Go to':
+      return translations.go_to
+    case 'Go to a HexDocs package':
+      return translations.go_to_hex
+    case 'Here are some tips when performing a full-text search:':
+      return translations.not_found_search_hint
+    case 'Keyboard Shortcuts':
+      return translations.keyboard_shortcuts
+    case 'License':
+      return translations.license
+    case 'macro':
+      return translations.macro
+    case 'module':
+      return translations.module
+    case 'Summary':
+      return translations.summary
+    case 'task':
+      return translations.task
+    case 'Toggle sidebar':
+      return translations.toggle_sidebar
+    case 'Toggle night mode':
+      return translations.toggle_night_mode
+    case 'Top':
+      return translations.top
+    case 'type':
+      return translations.type
+    case 'Types':
+      return translations.types
```

Comment:
Do you know where "type" and "Types" are used? I wonder if we could merge those together.

Reply:
So I added the 'type' because it is been used for the search result page inside parenthesis (this.type) :

assets/js/templates/search-results.handlebars
```
      <h2 class="result-id">
        <a href="{{this.ref}}">{{this.title}} <small>({{useTranslation this.type ../translations}})</small></a>
      </h2>
```

... and 'Types' is part of the sidebar and comes from the group.nodes.id

assets/js/templates/sidebar-items.handlebars
```
                {{#each group.nodes}}
                  <li>
                    <a href="{{node.id}}.html#{{anchor}}">{{{ useTranslation id ../../../translations }}}</a>
                  </li>
                {{/each}}
```